nvflare/fuel/hci/client/cli.py

```diff
@@ -22,6 +22,11 @@
 from enum import Enum
 from typing import List, Optional
 
+try:
+    import readline
+except ImportError:
+    readline = None
+
```

Comment:
Why do we have try except block here?

If readline is not built-in, we should make it a requirements and remove all the `if readline` statement so that we can support this cli history feature for all the users.

Reply:
readline is only for Unix, so this is to avoid causing errors for Windows or OS X

Reply:
I see that the readline module is not supported in Windows system.
In that case, we say we only support cli history feature for Unix and mac OS then?

Reply:
We can add a note later in the docstring in a separate PR, but we can merge this now.